modules/kit-elements-defaults/module.php

```diff
@@ -45,5 +48,7 @@ public function __construct() {
 		} );
 
 		Plugin::$instance->data_manager_v2->register_controller( new Controller() );
+
+		( new Usage() )->register();
```

Comment:
if is_admin

Reply:
I didn't find any other place in the code where we check that before sending the tracking data...